docker/rust/Dockerfile

```diff
@@ -1,18 +1,32 @@
-FROM gnosispm/dfusion-rust-base:v4
+ARG SOLVER_BASE=ubuntu:bionic
```

Comment:
This default value would be the right value for using the simple solver, right? Maybe a small comment would help. It took me some time to understand.